### PR TITLE
Reskin/menu UI

### PR DESCRIPTION
**Pull request by DarksightKellar**
This PR fixes a few minor UI inconsistencies on some menus around the app. Screenshots below.

First, the sort menu is more consistent with other menus from figma:
Screenshot 2024-05-01 at 18 23 05

Manage safe menu:
Screenshot 2024-05-01 at 18 33 17

Proposal filter menu
Screenshot 2024-05-01 at 18 35 00

Wallet menu (removed some corner artifacts):
Screenshot 2024-05-01 at 18 37 08

### CODE REVIEW

File: src/components/Proposals/index.tsx [hunk 1]
```diff
@@ -139,29 +138,43 @@ export default function Proposals() {
...
+            >
               <Button
                 variant="text"
                 paddingLeft={0}
                 paddingRight={0}
                 justifyContent="flex-start"
                 onClick={() => setFilters(allOptions)}
               >
-                <Text color="grayscale.100">{t('selectAll', { ns: 'common' })}</Text>
+                <Button
```
[the remark this answers is not in the document]
Comment (DarksightKellar) [reply src/components/Proposals/index.tsx:152]: Ah thanks for catching this. I didn't realise there was a Button parent. I've removed the inner button and restyled the parent to fit the same look

File: src/components/ui/utils/Sort.tsx [hunk 2]
```diff
@@ -67,18 +70,27 @@ export function Sort({ sortBy, setSortBy, buttonProps }: ISort) {
...
+          color="neutral-7"
+        >
+          {t('sortTitle')}
+        </Text>
         <SortMenuItem
           labelKey={SortBy.Newest}
           testId="sort-newest"
           onClick={() => setSortBy(SortBy.Newest)}
         />
         {/* TODO Divider look doesn't quite match */}
```
[the remark this answers is not in the document]
Comment (DarksightKellar) [reply src/components/ui/utils/Sort.tsx:92]: Yes, good catch

File: src/components/ui/menus/FavoritesMenu/Favorite.tsx [hunk 3]
```diff
@@ -34,10 +34,12 @@ export function Favorite({ network, address }: IFavorite) {
         justifyContent="flex-start"
         gap={2}
       >
-        <Star
-          size="1.5rem"
-          weight="fill"
-        />
+        {daoRegistryName && (
```
[the remark this answers is not in the document]
Comment (DarksightKellar) [reply src/components/ui/menus/FavoritesMenu/Favorite.tsx:37]: Man, I spent a lot of time trying to decide what a good "less janky" method would look like. Ideal case would be to show a short, empty box with a loader or placeholder while the DAO names are being retrieved.

Problem is that each DAO name is retrieved in parallel via individual `useDAOName` hooks. To achieve this ideal would require some communication outside _all_ the favourite DAOs, with which the menu component can use to resize itself and render all the DAOa _at the same time_ when ready.

Wasn't worth pursuing in this PR.

Oh but wait, I'm just realising that you're probably talking about a placeholder for _each_ name. Hmmmm 🤔

[the remark this answers is not in the document]
Comment (DarksightKellar) [reply src/components/ui/menus/FavoritesMenu/Favorite.tsx:37]: Niiiice. This looks muuuuch better. Thank you @mudrila !

Screenshot 2024-05-02 at 11 51 48

[the remark this answers is not in the document]
Comment (DarksightKellar) [reply src/components/ui/menus/FavoritesMenu/Favorite.tsx]: @nicolaus-sherrill requesting review from you as well to get your thoughts on this convo and result.

**Review by adamgall — APPROVED** [review]
Looking solid sir

**Pull request merged**